src/Core/AdminConsole/OrganizationFeatures/OrganizationUsers/Interfaces/IOrganizationUserUserDetailsQuery.cs

```diff
@@ -6,4 +6,8 @@ namespace Core.AdminConsole.OrganizationFeatures.OrganizationUsers.Interfaces;
 public interface IOrganizationUserUserDetailsQuery
 {
     Task<IEnumerable<OrganizationUserUserDetails>> GetOrganizationUserUserDetails(OrganizationUserUserDetailsQueryRequest request);
+
+    Task<IEnumerable<(OrganizationUserUserDetails, bool, bool)>> Get(OrganizationUserUserDetailsQueryRequest request);
+
+    Task<IEnumerable<(OrganizationUserUserDetails, bool, bool)>> GetConfirmed(OrganizationUserUserDetailsQueryRequest request);
```

Comment:
Given that we have no other uses for this right now, I'm OK with this being specific - I suggest `GetAccountRecoveryEnrolledUsers`. This also allows you to do more filtering earlier, and reduces the logic in the controller.

(Account Recovery is the updated name for this feature)